sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -37,6 +37,16 @@ export interface AddParticipantsRequest {
     participants: ChatParticipant[];
 }
 
+// @public
+export interface ChatAttachment {
+    attachmentType: "image";
```

Comment:
public api should be of type AttachmentType (even though it's not part of generated code)